app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>TaskList</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <header>
+    <h1 class="title">TASKLIST</h1>
+      <ul>
+        <li class="task" ><%= link_to "Add Task", new_task_path %></li>
+        <li class="task" ><%= link_to "All Tasks", tasks_index_path %></li>
```

Comment:
If we remove the route on line 2 in `config/routes.rb` then this route helper would need to be `tasks_path` instead of `tasks_index_path`, which is more in keeping with Rails conventions.